diegobusr/react-apprenticeship-capstone1
# Deliverable 3 added global context, useReducer and dark theme feature

## Description
```
- added global context
- added use of useReducer
- added dark theme feature with the toggle button on the header
```
## Type of Change

## How should this be tested?
```
- Verify that the fetch works well according to the input in the search bar
- Theme toggle button should work on every page of the app
```
## What is the deliverable?
```
3
```
